src/Tasks.UnitTests/AssemblyDependency/Miscellaneous.cs

```diff
@@ -3538,7 +3538,7 @@ public void ConflictBetweenNonCopyLocalDependencies()
 
             Execute(t);
 
-            Assert.Equal(2, t.ResolvedDependencyFiles.Length);
+            Assert.Equal(3, t.ResolvedDependencyFiles.Length);
```

Comment:
Is the new one here `G`, added for the new test? Please add an assert for it as "documentation".